modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="reviewing-cluster-status-from-the-openshift-cluster-manager_{context}"]
+= Reviewing the cluster status from the {cloud-redhat-com}
+
+You can review detailed information about the status of your cluster in the {cloud-redhat-com}.
```

Comment:
Should we drop the second "the" in this sentence. For example, we don't say "the OpenShift Container Platform", but not sure if the same should apply for Cluster Manager.

Reply:
Looking through the OpenShift docs library, most references are to "the Red Hat OpenShift Cluster Manager". Given this, I will keep this line as it is for now. If we decide that the "the" should be dropped, we should do so throughout the library, in my view.